Sample-App/app/src/main/java/com/adobe/marketing/mobile/sampleapp/EdgeConsentTab.java

```diff
@@ -0,0 +1,143 @@
+/*
+  Copyright 2020 Adobe
+  All Rights Reserved.
+
+  NOTICE: Adobe permits you to use, modify, and distribute this file in
+  accordance with the terms of the Adobe license agreement accompanying
+  it.
+ */
+
+package com.adobe.marketing.mobile.sampleapp;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+
+import androidx.fragment.app.Fragment;
+
+import com.adobe.marketing.mobile.AdobeCallback;
+import com.adobe.marketing.mobile.edge.consent.Consent;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * create an instance of this fragment.
+ */
+public class EdgeConsentTab extends Fragment implements NavigationAware {
+    private static final String LOG_TAG = "EdgeConsentTab";
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_edgeconsent_tab, container, false);
+    }
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        // Card view to display registered Consent extension version
+        final TextView tv_version  = getView().findViewById(R.id.tv_edgeConsent_version);
+        view.post(new Runnable() {
+            @Override
+            public void run() {
+                if (tv_version != null) {
+                    tv_version.setText("Edge Consent Extension version: " + Consent.extensionVersion());
```

Comment:
First of all, thank you for this change!!!! 

can be simplified to "Consent v" + Consent.extensionVersion